spec/controllers/storage_controller_spec.rb

```diff
@@ -73,6 +73,9 @@
         login_as FactoryGirl.create(:user, :features => button)
 
         host = FactoryGirl.create(:host)
+        command = button.split('_', 2)[1]
+        Host.any_instance.stub(:is_available?).with(command).and_return(true)
```

Comment:
@Fryguy : what subclass do you mean?

```
[10] pry(#<RSpec::Core::ExampleGroup::Nested_1::Nested_1>)> FactoryGirl.create(:host_vmware_esx).is_available?('reboot')
=> false
[11] pry(#<RSpec::Core::ExampleGroup::Nested_1::Nested_1>)> FactoryGirl.create(:host_vmware).is_available?('reboot')
=> false
[12] pry(#<RSpec::Core::ExampleGroup::Nested_1::Nested_1>)> FactoryGirl.create(:host_redhat).is_available?('reboot')
=> false
```

looking at `app/models/host.rb` it makes the impression to me that real stuff has to connected for these methods to return true